awx/sso/tests/unit/test_fields.py

```diff
@@ -114,3 +115,23 @@ def test_internal_value_invalid(self, group_type, data, expected):
         with pytest.raises(ValidationError) as e:
             field.to_internal_value(data)
         assert e.value.detail == expected
+
+
+class TestLDAPServerURIField():
+
+    valid_URIs = [
+        r'ldap://servername.so3:444',
+        r'ldaps://servername3.s300:344'
+    ]
+
+    invalid_URIs = [
+        r'ldaps://servername.-so3:444'
+    ]
```

Comment:
Could you use the pytest parameterization for these? like:

https://github.com/ansible/awx/blob/d9965cfe7e0f7a82fea04acb3b6b49852874f655/awx/main/tests/unit/utils/test_filters.py#L15

that helps to make it clear which case the test failed on if it fails.

Reply:
thanks for the suggestion, I changed the tests to include pytest.mark.parametrize()